src/Handler/CurlFactory.php

```diff
@@ -15,8 +15,8 @@
  */
 class CurlFactory implements CurlFactoryInterface
 {
-    const CURL_VERSION_STR = 'curl_version';
-    const LOW_CURL_VERSION_NUMBER = '7.21.2';
+    protected const CURL_VERSION_STR = 'curl_version';
+    protected const LOW_CURL_VERSION_NUMBER = '7.21.2';
```

Comment:
Made this protected since class is not final someone might need them if he extends `CurlFactory` for whatever reason

Reply:
Why not public?

Reply:
I don't have a good argument.. So let's make them public and maybe we can revise for the visibility on v8

Reply:
I don't have a good argument.. So let's make them public and maybe we can revise for the visibility on v8

Reply:
Done.